tests/test_src.py

```diff
@@ -1,25 +1,21 @@
 from app import set_prices
-from app import app
+
 
 def test_set_prices_sorting_works():
-    #
+    # Sortowanie
     prices = [10, 30, 20]
     prices.sort()
-   # return prices
 
-    #
+    # Sorotwanie z mnożnikiem
     result = set_prices(prices, 1.0)
 
-
-    # if result_sorted != result:
-    #     raise AssertionError()
+    # Sprawdzenie czy działa
     assert prices == result
     assert prices == [10, 20, 30]
 
 
 def test_set_prices_multiplying_works(prices=[10, 20, 30], a=2):
 
```

Comment:
I think that 'prices' and 'a' should belong to unit-test code not parameters. Please move these two below

Reply:
still waiting :)